dvc/remote/gdrive.py

```diff
@@ -109,8 +109,11 @@ def init_drive(self):
     def gdrive_upload_file(
         self, args, no_progress_bar=True, from_file="", progress_name=""
     ):
+        param = {"supportsAllDrives": True}
```

Comment:
I think we can keep `support*Drivers` family of params inside? They are deprecated anyway and will be enabled by default soon? Also were they present even before we forked the PyDrive?

Reply:
In original PyDrive from pip some support of related params was added ( deprecated `supportsTeamDrives` was used ), but some bugs were fixed in PyDrive's master and not released to pip.

> I think we can keep support*Drivers family of params inside? 
Sounds good.